integrations/tensorflow/iree_tf_compiler/BUILD.bazel

```diff
@@ -71,24 +71,10 @@ cc_binary(
     ],
 )
 
-cc_binary(
+py_binary(
```

Comment:
Just delete this. There is no need to "build" a binary for Python. We have elected to completely bail on trying to use Bazel for Python dependency management. It may be desirable to move the tool into a separate directory so that it's clearly distinct from these compile C++ binaries (which should also go away soon). Then there are no symlinks: the CMake build just directly references the Python executable script.

Reply:
(I assume this is a slightly delayed comment, so resolving. I removed this in the previous commit.)